ios/brave-ios/Sources/BraveWallet/Preview Content/MockKeyringService.swift

```diff
@@ -118,7 +118,7 @@ class MockKeyringService: BraveWalletKeyringService {
     completion(isBackedUp)
   }
 
-  func mnemonicForDefaultKeyring(password: String, completion: @escaping (String) -> Void) {
+  func walletMnemonic(password: String, completion: @escaping (String) -> Void) {
```

Comment:
Need optional string in completion closure now, same in `createWallet` function.
```suggestion
  func walletMnemonic(password: String, completion: @escaping (String?) -> Void) {
```